src/map.rs

```diff
@@ -248,6 +252,30 @@ impl Map {
         Self::draw_killfeed(canvas, assets, game_state)
     }
 
+    fn draw_all_hurricanes(
```

Comment:
Maybe rename this function to draw_hurricane(...), and the other draw_hurricane(...) to draw_hurricane_part(...) or draw_hurricane_block(...)